pages/_app.tsx

```diff
@@ -1,14 +1,29 @@
 import App from 'next/app';
 import React from 'react';
+import { Provider } from 'react-redux';
+import { applyMiddleware, compose, createStore } from 'redux';
+import thunk from 'redux-thunk';
 import theme, { themeObject } from 'styles/theme';
 import { ThemeProvider as MaterialUIThemeProvider } from '@material-ui/core/styles';
```

Comment:
This was done by @nik-john but just to be consistent could you please order imports since you're already working on this file?
1. npm libs (`react`, `next`, `@material-ui`)
2. our packages (`styles`, `components`)
3. local depenedcies (none in this case but `./someRandomComponent`)

I'll add this to the docs later

Reply:
I haven't gone through that part of documentation, but will surely update my branch accordingly.

Reply:
Fixed it. :100: